lambdas/initialise-ipv-session/src/test/java/uk/gov/di/ipv/core/initialiseipvsession/InitialiseIpvSessionHandlerTest.java

```diff
@@ -157,6 +157,7 @@ static void setUpBeforeAll()
 
     @BeforeEach
     void setUp() throws Exception {
+        claimsBuilder.claim(VTR, List.of("Cl.Cm.P2", "Cl.Cm.PCL200"));
```

Comment:
This is because there's a parameterised test which changes the vtr claims so we need to reset them here